docs/en/integrations/data-ingestion/etl-tools/dbt/dbt-incremental-model.md

```diff
@@ -194,72 +193,27 @@ Adjust the above query to the period of execution. We leave result inspection to
 
 
 1. The plugin creates a temporary table `actor_sumary__dbt_tmp`. Rows that have changed are streamed into this table.
-    ```sql
-    create table actor_summary__dbt_tmp
-        engine = MergeTree()
-        order by ((updated_at, id, name))
-    as (with actor_summary as (
-       SELECT id,
-          any(actor_name) as name,
-          uniqExact(movie_id)    as num_movies,
-          avg(rank)                as avg_rank,
-          uniqExact(genre)         as genres,
-          uniqExact(director_name) as directors,
-          max(created_at) as updated_at
-       FROM (
-           SELECT imdb.actors.id as id,
-               concat(imdb.actors.first_name, ' ', imdb.actors.last_name) as actor_name,
-               imdb.movies.id as movie_id,
-               imdb.movies.rank as rank,
-               genre,
-               concat(imdb.directors.first_name, ' ', imdb.directors.last_name) as director_name,
-               created_at
-           FROM imdb.actors
-               JOIN imdb.roles ON imdb.roles.actor_id = imdb.actors.id
-               LEFT OUTER JOIN imdb.movies ON imdb.movies.id = imdb.roles.movie_id
-               LEFT OUTER JOIN imdb.genres ON imdb.genres.movie_id = imdb.movies.id
-               LEFT OUTER JOIN imdb.movie_directors ON imdb.movie_directors.movie_id = imdb.movies.id
-               LEFT OUTER JOIN imdb.directors ON imdb.directors.id = imdb.movie_directors.director_id
-       )
-       GROUP BY id
-    )
-     
-    select *
-    from actor_summary
-    where id > (select max(id) from imdb_dbt.actor_summary) or updated_at > (select max(updated_at) from imdb_dbt.actor_summary));
-    ```
+2. A new table `actor_summary_new` is created. The rows from the old table are, in turn, streamed from the old to new, with a check to make sure row ids do not exist in the temporary table. This effectively handles updates and duplicates.
+3. The results from the temporary table are streamed into the new `actor_summary` table:
+4. Finally, the new table is exchanged atomically with the old version via an `EXCHANGE TABLES` statement. The old table is inturn deleted.
```

Comment:
typo -- `inturn`.

Maybe should mention both the temporary and old table are dropped (not deleted)